feature.env

```diff
@@ -2,9 +2,13 @@ PORT=3007
 HMPPS_AUTH_URL=http://localhost:9091/auth
 TOKEN_VERIFICATION_API_URL=http://localhost:9091/verification
 TOKEN_VERIFICATION_ENABLED=true
+
 EDUCATION_AND_WORK_PLAN_API_URL=http://localhost:9091
 PRISONER_SEARCH_API_URL=http://localhost:9091
+CURIOUS_API_URL=http://localhost:9091
+DPS_URL=https://digital-dev.prison.service.justice.gov.uk
 NODE_ENV=development
+
```

Comment:
The env file for `start-feature` (which is used by our cypress integration tests) was missing a couple of entries, notably for this PR the endpoint for curious API pointing at wiremock